ci/maven-publish.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Maven Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
Each workflow has `runs-on: ubuntu-latest` with the exception of `swift.yml`. I changed a few workflows so that the spacing is more consistent, but I think a the same comment on each workflow is a little redundant. It makes sense for `blank.yml` as new users who are unfamiliar with the syntax will be able to learn what it is, but for other workflows it's seems excessive